addpokemon.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require_once "collectionfunctions.php";
+require_once "collectiondb.php";
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Pokemon Collection</title>
+	<link rel="stylesheet" href="normalize.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+<body>
+<h1>Add Pokemonz</h1>
+<form method="post">
+    <label for="name">Name <input type="text" name="name" required /></label>
```

Comment:
Not sure what is best practice but I've only ever seen inputs outside of label tags: e.g. 
<label for="thing >Thing</label>
<input name="thing" type="" />

Reply:
Best practice is to have input inside label tag